src/scala/ly/stealth/mesos/kafka/Executor.scala

```diff
@@ -92,6 +95,25 @@ object Executor extends org.apache.mesos.Executor {
       }
     }
 
+    def startCollectingMetrics(): Unit = {
+      new Thread {
```

Comment:
This might be better done with an Akka actor or Actor System (pool) for a few reasons.

1) it would be nice for a framework message to be received with more/less metrics become requested from the brokers. A scenario for this is you start to have some issue so you want to look at 1 topics metrics (for a few minutes). You can then just pass in the metric you want to see and then a few seconds later poll GET for status about it. There are _a_lot_ of kafka metrics and sometimes you want to just see quick inside the stream of them for what is going on. Helpful too to help validate downstream issues at times.

2) we can have knowledge of the health of this operation better to guard against bad/invalid metrics causing issues and not excepting in thread running that we can't catch or manage, etc

doesn't have to be akka but do not like the new Thread approach here without guarding better (which systems like Akka make it easier to-do it with